Changed Color of Game

LF-60730 opened the pull request:
Created a new color, cherry and added it to constants, then replaced every instance of the RED tuple with it to change the color of the board and pieces.

LF-60730 committed:
Changed Color of Game

Created a new color, cherry and added it to constants, then replaced every instance of the RED tuple with it to change the color of the board and pieces.

tun67213 reviewed (approved):
The changes suggested seem to be changes that are reasonable and visible, but not too much of a change. Therefore, I approve of these changes.

LF-60730 committed:
Added Basic Graphics for Twitter Button

Drew the button to fetch tweets in the checkers game

ewongzz committed:
Changed color theme, added fonts, switched settings and leaderboard position

sharrontum committed:
Merge pull request #1 from LF-60730/ernest

Task 1

LF-60730 committed:
Added basic display box for tweets and fixed twitter button

Finished drawing the buttons and text box for displaying tweets

LF-60730 committed:
Switched to reddit and managed initial access

Decided to access reddit posts instead of twitter posts, and print out the topmost post to the terminal.

LF-60730 committed:
Able to print out heading of reddit post

Wrote functions in game.py to handle fetching heading of most recent reddit post and display it, and altered game loops in secondmenu.py to run these functions during gameplay.

LF-60730 committed:
Added final comments

Added some comments to explain new functions that handle fetching and displaying title of latest post on temple subreddit

LF-60730 committed:
Merge branch 'main' into Lara

sharrontum committed:
Merge pull request #2 from LF-60730/Lara

Lara

sharrontum committed:
Added Quit Feature

sharrontum committed:
Added feature to quit mid-game

sharrontum committed:
Add feature to quit mid-game

sharrontum committed:
Add quit feature

LF-60730 committed:
Merge pull request #3 from LF-60730/sharron

Sharron

LF-60730 committed:
Finished adding Difficulty levels

Added different levels of difficulty for playing against computer, so user can adjust time for turn.
